.github/CHANGELOG.md

```diff
@@ -51,6 +51,9 @@
 * Add a Catalyst-specific wrapping class for Lightning Kokkos.
   [(#770)](https://github.com/PennyLaneAI/pennylane-lightning/pull/770)
 
+* Update the Catalyst-specific wrapping class for Lightning Kokkos to track Catalyst's new support for MCM seeding.
+  [(819)](https://github.com/PennyLaneAI/pennylane-lightning/pull/819)
```

Comment:
Please add the new entry to the top of the list.

Reply:
done